llama_stack/providers/inline/huggingface/datasetio/huggingface.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+from typing import List, Optional
+
+from llama_stack.apis.datasetio import *  # noqa: F403
+
+
+from datasets import Dataset, load_dataset
```

Comment:
this is huggingface library? can we namespace this import as `import datasets as huggingface_datasets` and use later? otherwise it is confusing because we also have these names.

Reply:
done!